### PR TITLE
Sort Option when Searching Repositories

### PR DESCRIPTION
**Pull request by kbabetas**
Added the option to sort repositories (By Best Match, Most Stars, Least Stars, Recently Updated, Least Recently Updated, Most Forks, Least Forks) when searching for a repo. Followed the ui design and the query method that is used in the other search functions and the final result looks like this:
56980267-93245e00-6b84-11e9-94e4-493ed2a01e79

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="sort_by">Sort By</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values/strings.xml]: This should be removed.

Comment (kbabetas) [reply app/src/main/res/values/strings.xml]: You're right I removed it!
Yes, I followed the way you implemented the other options and then I only needed to find the right queries.

**Comment by kbabetas** [comment]
@k0shk0sh do I need to do something to resolve the failing checks at bitrise or is it something that you can resolve?

**Comment by yakov116** [comment]
@k0shk0sh the key store needs to be added for pull requests

**Comment by k0shk0sh** [comment]
Probably, I'll add that later.

**Pull request closed without merging**